lib/datahub-client/tests/client/datahub/test_datahub_client.py

```diff
@@ -389,15 +392,15 @@ def test_get_dataset(
                         entity_type="Database",
                     )
                 ],
-                RelationshipType.DATA_LINEAGE: [],
             },
             domain=DomainRef(display_name="", urn=""),
             governance=Governance(
                 data_owner=OwnerRef(display_name="", email="", urn=""),
                 data_stewards=[],
             ),
             tags=[TagRef(display_name="some-tag", urn="urn:li:tag:Entity")],
-            last_modified=datetime(2024, 3, 5, 6, 16, 47, 814000, tzinfo=timezone.utc),
+            last_modified=1709619407814,
+            provider="LAA",
```

Comment:
we've still got provider here, is this being used? (I was assuming we were leaving this out for now)